tests/general/InvalidLinkBearTest.py

```diff
@@ -77,6 +77,22 @@ def assertResult(self, valid_file=None, invalid_file=None, settings={}):
                 self.assertNotEqual(out, [])
                 self.assertNotEqual(out, None)
 
+    def assertResultCountPerLine(self, line, expected_num):
```

Comment:
The name suffix  '....PerLine' does not make sense.
There is only one line.
Also it is strange that `line` is an array.

Either the mentod should only accept a single line, as a `str`, with one expected line count.
Or, accept and process multiple lines, and compare against a list of expected line counts.